schemas/telemetry/configuration-schema.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "telemetry/configuration-schema.json",
+  "title": "TelemetryConfigurationEvent",
+  "type": "object",
+  "description": "Schema of all properties of a telemetry configuration event",
+  "allOf": [
+    {
+      "$ref": "_common-schema.json"
+    },
+    {
+      "required": ["telemetry"],
+      "properties": {
+        "telemetry": {
+          "type": "object",
+          "description": "The telemetry configuration information",
+          "required": ["type", "configuration"],
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "Telemetry type",
+              "const": "configuration"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "Configuration properties",
+              "required": ["session_sample_rate", "telemetry_sample_rate", "trace_sample_rate", "use_proxy"],
+              "properties": {
+                "session_sample_rate": {
+                  "type": "integer",
+                  "description": "The percentage of sessions tracked",
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "telemetry_sample_rate": {
+                  "type": "integer",
+                  "description": "The percentage of telemetry events sent",
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "trace_sample_rate": {
+                  "type": "integer",
+                  "description": "The percentage of requests traced",
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "premium_sample_rate": {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏let's use session replay sample rate since premium is deprecated